DungeonEnemies.lua

```diff
@@ -307,8 +307,9 @@ local function setUpMouseHandlersAwakened(self,clone,scale,riftOffsets)
 end
 
 function DungeonToolsEnemyMixin:OnClick(button, down)
+    local mapEnemiesLocked = db.mapEnemiesLocked
```

Comment:
Persisting this in the Ace DB may not be a good idea; it means that if somebody logs out and comes back, they'll be locked.

Locking per-route may be a better approach, or using an in-module `local` to achieve this (so the add-on unlocks on reload)